lib/Account.php

```diff
@@ -33,6 +33,7 @@
 
 namespace OCA\Mail;
 
+use Horde\ManageSieve;
```

Comment:
If you add `composer.lock` this warning should go away.

Reply:
Wouldn't that mess with dependabot?

Reply:
No, that is fine. If a dependency changes both the .json and .lock files should be committed